src/data/index.tsx

```diff
@@ -19,7 +20,15 @@ export const data: Category[] = [
     name: 'Page Sections',
     subLabel: 'Large Page Sections like Hero, Features ...',
     id: 'page-sections',
-    children: [hero, features, footer, testimonials, pricing, statistics],
+    children: [
+      hero,
+      features,
+      footer,
+      testimonials,
+      pricing,
+      statistics,
+      cookiePreferences,
+    ],
```

Comment:
I think that does not fit really into the Page  `Sectionts category`. I would move it into `Cards`

Reply:
okay i will move it into cards